.travis.yml

```diff
@@ -16,7 +16,6 @@ addons:
     - liblapack-dev
 
 env:
-    - TEST_DIR=tests/examples
     - TEST_DIR=tests/em/fdem/inverse/derivs
```

Comment:
I am no longer testing the examples. They are being tested inside of the doc tests, but will not go towards coverage.